TestData/CharacterizationTestSolution/VbNetStandardLib/VbNetStandardLib.vbproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>VbNetStandardLib</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Since this needs nuget packages to build it fails on the build server.
* [ ] Add code to the test to restore nuget packages so that we can properly test this